### ISSUE TITLE
Problem with the previous and next buttons (version 5.3)

### ISSUE THREAD

**Issue by aurel85** [PyvesB/AdvancedAchievements#251]
Hi,

I made the 5.3 update of the plugin. The op and the players fail to show all the achievements because the previous and next buttons do not work. I can not display more than 5 achievements when there are 6 or 7 for example. There is no error message in the console.

At no time in the config the unrealized achievements are masked (and anyway even the achievements achieved can not be seen) and even making a complete reset of the config that did not change anything.

Am I the only one with this problem ?

I use PaperSpigot 1.12.2.

**Comment by PyvesB** [comment]
Hello aurel85,

Sorry about that. Unfortunately the pagination didn't get as much testing as I would have wanted it to, as the people who were nagging me for that feature still haven't given any feedback even though I shared my first test version well before the official release. Never mind, here's another version, could you please tell me whether it helps? :wink:

[AdvancedAchievements.zip](https://github.com/PyvesB/AdvancedAchievements/files/1428471/AdvancedAchievements.zip)

Cheers,

Pyves

**Comment by aurel85** [comment]
With this version all my achievements appear without the slightest problem, but the previous and next buttons are useless. For example in this section I have 17 successes and everything is displayed :

https://sandbox.fr/images/uploads/original/1509392177.png

Is this normal ? Because currently the buttons are not useful anymore, if so, would it be possible that they do not appear ? Or should it show 5 successes so that I can scroll through with the 2 buttons (as should have been the case on version 5.3) ?

**Comment by PyvesB** [comment]
I have removed the previous and next buttons if you do not have enough achievements (= 50) to fill in one page for a given category. Nevertheless, if you have over 50 achievements for the category, both buttons will systematically appear (basically you'll be in pagination mode). Does this better suit your needs? I have attached a new test file.

[AdvancedAchievements.zip](https://github.com/PyvesB/AdvancedAchievements/files/1428808/AdvancedAchievements.zip)
